user-guide/phase2.rst

```diff
@@ -0,0 +1,77 @@
+Phase 2 - XC50 ARM
+------------------
+
+Cray XC50 cabinet containing 10'000 ARM cores, available since November 2018.
+
+* Cavium ThunderX2 ARM v8.1
+
+  * 16nm FinFET CPU @ 2.2GHz
```

Comment:
Ours should be running at 2.1 GHz (unless something has changed!).

Reply:
```suggestion
  * 16nm FinFET CPU @ 2.1GHz
```